### PR TITLE
Polish UX for `rzup`

### PR DESCRIPTION
**Pull request by nuke-web3**
Suggestion for better output in `rzup`.

### CODE REVIEW

File: rzup/README.md [hunk 1]
```diff
@@ -4,9 +4,8 @@ Install, update, or revert to a specific RISC Zero version.
 
 ## Install
 
-<!-- TODO: Replace this friendly redirect URL once set up -->
 ```sh
-curl -L https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/install | bash
+curl -L https://raw.githubusercontent.com/risc0/risc0/main/rzup/install | bash
```
Comment (nuke-web3) [inline rzup/README.md:8]: Not sure if there was a plan for some other link, but IMHO github should be used over aws.

**Comment by flaub** [comment]
The issue here is that GitHub has IP rate limiting. The chance of this goes up a lot due to for example CI builds coming from a single IP (like in a cluster in the cloud).

**Comment by flaub** [comment]
Ooops, just saw that there are more changes than just the URL, re-opening

File: rzup/README.md (hunk 1 above)
Comment (flaub) [reply rzup/README.md:8]: We will likely setup a more permanent link, but it will be hosted in a bucket rather than on GitHub.

**Comment by nuke-web3** [comment]
Ah so we do want to use some other URL, but need devops to configure a risczero.com or other redirect?

I am happy to include only the suggested UX polish in this PR if desired or close it.

File: rzup/install [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+
+echo "💾 Installing rzup"
 
 BASE_DIR=${XDG_CONFIG_HOME:-$HOME}
 RISC0_DIR=${RISC0_DIR-"$BASE_DIR/.risc0"}
 RISC0_BIN_DIR="$RISC0_DIR/bin"
 
-BIN_URL="https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/rzup"
+# TODO: Replace this friendly redirect URL once set up
+BIN_URL="https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/install"
```
Comment (flaub) [inline rzup/install:16]: Note, this is the wrong URL
```suggestion
BIN_URL="https://risc0-artifacts.s3.us-west-2.amazonaws.com/rzup/rzup"
```

Comment (nuke-web3) [reply rzup/install]: Doh! Copy pasta fail

**Review by flaub — APPROVED** [review]

**Pull request merged**